ray_julia_jll/test/function_descriptor.jl

```diff
@@ -1,3 +1,5 @@
+using ray_julia_jll: JuliaFunctionDescriptor, function_descriptor
```

Comment:
Should we move this to `runtests.jl`?

Reply:
that's where it was, but other test files have their own `using`s and it seemed weird to have ONLY these imports at the top level